JamGame/Assets/Scripts/TileBuilder/TileBuilderImpl.cs

```diff
@@ -388,5 +403,42 @@ public BuildingConfig SaveBuildingIntoConfig()
 
             return buildingConfig;
         }
+
+        public List<CoreModel> GrowMeetingRoom(MeetingRoomLogics meetingRoom, int growCount)
```

Comment:
1. pass `growCount` into `meetingRoom.GetMeetingRoomGrowingInformation()` and don't use `for` loop here
2. move this logics into controller